website/templates/project/registrations.mako

```diff
@@ -22,23 +30,147 @@
+                  </div>
+                  <div class="col-md-1">
+                    <!-- TODO: uncomment for exposing registration approval logic 
+                    <a class="btn btn-success" data-bind="attr.href: urls.register,
+                                                          tooltip: {
+                                                            placement: top, 
+                                                            title: isApproved ? 'Finialize this draft' : 'This draft must be approved before it can be registered'
+                                                          },
+                                                          css: {'disabled': !isApproved}">Register</a>
+                    -->
+                    <a class="btn btn-success" data-bind="attr.href: urls.register_page">Register</a>
+                  </div>
+                </div>                
+              </h4>
+            </li>
+          </div>
+        </div>
+      </div>
+      <div data-bind="if: preview">
+        <br />
+        <button data-bind="click: preview.bind($root, false)"
+                class="btn btn-primary"><i class="fa fa-arrow-circle-o-left"></i>&nbsp;&nbsp;&nbsp;Back</button>
+        <br />
+        <br />
+        <p> Select a registration template to continue ... </p>
+        <div class="row">
+          <form name="createDraft" method="post">
+            <div class="col-md-9">
+              <div class="form-group">
+                <select class="form-control" data-bind="options: schemas,
+                                                        optionsText: 'name',
+                                                        value: selectedSchema">
+                </select>
+                <input type="hidden" name="schema_name" data-bind="value: selectedSchema().name" />
+                <input type="hidden" name="schema_version" data-bind="value: selectedSchema().version" />
+              </div>          
+            </div>
+            <div class="col-md-3">
+              <button type="submit" class="btn btn-success"> Start </button>
+            </div>
+          </form>
+        </div>
+        <hr />
+        <div class="row" data-bind="if: selectedSchema">
+          <div class="col-md-12" data-bind="with: selectedSchema">
+            <span data-bind="if: schema.config">
+              <h4> Fulfills: </h4>
+              <div class="row">
+                <div class="col-md-12 schema-fulfillment" data-bind="foreach: schema.config.fulfills">
+                  <span class="well">
+                    <span data-bind="text: name"></span>&nbsp;&nbsp;
+                    <a class="fa fa-info-circle" target="_blank" data-bind="attr.href: info"></a>
+                  </span>
+                </div>
+              </div>
+            </span>
+            <h4> Description: </h4> 
+            <blockquote>
+              <p data-bind="html: schema.description"></p>
```

Comment:
Use data-bind=text instead of html.

Bind as HTML only as a last resort, and then only for text that the user can never possibly modify.